server/middleware/setUpGovukOneLogin.ts

```diff
@@ -47,8 +47,6 @@ export default function setUpGovukOneLogin(): Router {
       } else res.redirect(client.endSessionUrl())
     })
 
-    router.use(govukOneLogin.authenticationMiddleware())
```

Comment:
Moving this to be called in `app.ts` so routes can be called before/after it to allow for unauthenticated access